docs/AddingNewProjects.md

```diff
@@ -0,0 +1,31 @@
+# Adding New Projects To The Repo
+
+Sample PR of final result: https://github.com/dotnet/aspnetcore/pull/41945
+
+## Creating a new project
+1. Create a new folder that will house your `.csproj` and other project-related files.
```

Comment:
Worth noting here that if this is a new folder in https://github.com/dotnet/aspnetcore/tree/main/src, you need to add a reference to it to eng\build.props:

https://github.com/dotnet/aspnetcore/blob/41e03fe5995cd59213056179343df168228650ce/eng/Build.props#L160-L168

And

https://github.com/dotnet/aspnetcore/blob/41e03fe5995cd59213056179343df168228650ce/eng/Build.props#L211-L219

Reply:
Otherwise build.cmd won't know to build it. Your new project is inside of `src\Middleware`, so it's already included automatically & you didn't need to do this step.